files/docker.wpt.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=WPT Container
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker swarm leave --force
+ExecStartPre=-/usr/bin/docker swarm init --advertise-addr 172.30.1.5
```

Comment:
I wanted to ensure the docker swarm was clean when restarting the service so I force docker to leave the existing swarm (this kills it since it only has 1 node) and start a new one.

`--advertise-addr` is hardcoded to match the vagrant box and would need to be parametrized in a real script.